mm2src/mm2_rpc/src/mm_protocol.rs

```diff
@@ -1,27 +1,11 @@
 use common::{HttpStatusCode, SerializationError};
 use http::{Response, StatusCode};
 use mm2_err_handle::prelude::*;
-use serde::{Deserialize, Serialize};
+use serde::Serialize;
 use serde_json as json;
 use serde_json::Value as Json;
 
-/// Please note there is no standardized `1.0` version, so this enumeration should not be used in the legacy protocol context.
-#[derive(Clone, Copy, Deserialize, Serialize)]
-pub enum MmRpcVersion {
-    #[serde(rename = "2.0")]
-    V2,
-}
-
-#[derive(Deserialize)]
-#[serde(deny_unknown_fields)]
-pub struct MmRpcRequest {
-    pub mmrpc: MmRpcVersion,
-    pub userpass: Option<String>,
-    pub method: String,
-    #[serde(default)]
-    pub params: Json,
```

Comment:
This default is also removed in the struct after it was moved, ref. https://github.com/KomodoPlatform/komodo-defi-framework/pull/1932#discussion_r1290801438
https://github.com/KomodoPlatform/komodo-defi-framework/blob/001abfffbf55e7e82e4c1ce20b9b0083c528d780/mm2src/mm2_rpc/src/data/version2.rs#L15-L24

Reply:
:fearful: :pray: it's very bad of mine, thank you for pointing it out! Sorry about this lack
[done](https://github.com/KomodoPlatform/komodo-defi-framework/pull/1932/commits/c4d661d725ddfc013c9fd7f5eb29f69988a5315f)